pandas/core/series.py

```diff
@@ -2333,12 +2354,16 @@ def combine(self, other, func, fill_value=None):
 
     def combine_first(self, other):
         """
-        Combine Series values, choosing the calling Series's values
-        first. Result index will be the union of the two indexes
+        Combine Series values, choosing the calling Series's values first.
+
+        Notes
```

Comment:
Can you reorder the sections as they are defined in https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
Looks good to me, @datapythonista , according to the NumPy docs.
Do you want me to shift the `See Also` section below `Examples`?